modules/library/render/src/main/java/org/geotools/renderer/lite/StreamingRenderer.java

```diff
@@ -1185,6 +1146,26 @@ Query getStyleQuery(FeatureSource<FeatureType, Feature> source,
         
         return query;
     }
+
+    private void setMetaBuffer(List<LiteFeatureTypeStyle> styleList, int metaBuffer) {
+        for (LiteFeatureTypeStyle fts : styleList) {
+            fts.metaBuffer = metaBuffer;
+        }
+
```

Comment:
I am not used to blank lines before a closing brace, seeing it a couple times in this patch and wanted to ask if it was on purpose.